spec/factories/bishop.rb

```diff
@@ -1,4 +1,7 @@
 FactoryGirl.define do
   factory :bishop do
+    player_id 1
```

Comment:
How do you know this player exists? You should use an association https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md